jaeger-zipkin/src/main/java/com/uber/jaeger/senders/zipkin/ZipkinSender.java

```diff
@@ -63,17 +64,17 @@ public static ZipkinSender create(String endpoint) {
    * sender = ZipkinSender.create(KafkaSender.create("192.168.99.100:9092"));
    * }</pre>
    */
-  public static ZipkinSender create(zipkin.reporter.Sender<byte[]> delegate) {
+  public static ZipkinSender create(zipkin.reporter.Sender delegate) {
```

Comment:
sender has no generic parameter anymore after we made encoders very fast https://github.com/openzipkin/zipkin/releases/tag/1.8.0